salt/modules/virt.py

```diff
@@ -710,27 +707,24 @@ def init(name,
         salt 'hypervisor' virt.init vm_name 4 512 nic=profile disk=profile
     '''
     hypervisor = __salt__['config.get']('libvirt:hypervisor', hypervisor)
-    log.debug('Using hyperisor {0}'.format(hypervisor))
+    log.debug('Using hyperisor %s', hypervisor)
 
     nicp = _nic_profile(nic, hypervisor, **kwargs)
-    log.debug('NIC profile is {0}'.format(nicp))
+    log.debug('NIC profile is %s', nicp)
 
     diskp = _disk_profile(disk, hypervisor, **kwargs)
 
     if image:
         # If image is specified in module arguments, then it will be used
         # for the first disk instead of the image from the disk profile
         disk_name = next(six.iterkeys(diskp[0]))
-        log.debug('{0} image from module arguments will be used for disk "{1}"'
-                  ' instead of {2}'
-                  .format(image,
-                          disk_name,
-                          diskp[0][disk_name].get('image', None)))
+        log.debug('%s image from module arguments will be used for disk "%s"'
+                  ' instead of %s', image, disk_name, diskp[0][disk_name].get('image', None))
```

Comment:
`None` is implied if no argument is passed to a `dict.get()`. This doesn't necessarily need to be changed, though.